model/Hierarchy.php

```diff
@@ -38,6 +38,19 @@ class Hierarchy extends DataExtension {
 	 */
 	private static $node_threshold_leaf = 250;
 
+	/**
+	 * A list of classnames to exclude from display in the hierarchy.
+	 * Especially useful for big sets of pages like listings
+	 * If you use this, and still need the classes to be editable
+	 * then add a model admin for the class
+	 * Note: Does not filter subclasses (non-inheriting)
+	 *
+	 * @var array
+	 * @config
+	 */
+	private static $hide_from_hierarchy = array();
+	private static $hide_from_cms_tree = array();
```

Comment:
Please split PHPdocs for these. The second var will end up not actually documented.